package.json

```diff
@@ -7,7 +7,9 @@
     "build": "lerna run build",
     "lint": "eslint packages --ext .ts,.tsx,.vue --quiet --cache --cache-strategy content",
     "lint:fix": "eslint packages --ext .ts,.tsx,.vue --fix",
-    "test": "lerna run test --parallel",
+    "test:unit": "jest",
```

Comment:
I see that running this locally it creates a file `rootDir/output/en.messges.json` but I'm still not sure why